### DISCUSSION TITLE
part-ii/2017-123/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#204] [Question comments]
Part II 2017 Probability and Measure: Paper 1, Section II, J 

https://questions.tripos.org/part-ii/2017-123/

**Comment by eha1234** [comment]
I got 0 for (e)

**Comment by ArthurConmy** [comment]
I did d) and e) by showing $f^{-1}( \{ n \} )$ was the countable union of "diagonal sets", sets defined by $ x_0,y_0,s \in [0,1) $:

$$ D_{ x_0,y_0,s } = \{ ( x,y ) : x_0 \le x \le x_0+s , y_0 \le y \le y_0+s \}. $$

These can be approximated by $m$ squares of side length $1 / m$ for any $m \in \mathbb{ Z}$, so have measure 0, and hence I get measure 0 for e) too with countable additivity.
